CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing / Hacking
+
+## Intended use case
+
+The Snap in this repository is specifically developed for the 
+[Magma](https://www.magmacore.org/) usecase.
+
+## Developing and testing
+Testing of the Snap is done using `tox`. To run tests, from this repositorie's main directory 
+execute one of the following commands:
+```shell
+tox -e lint      # code style
+tox -e static    # static analysis
+tox -e unit      # unit tests
+```
+To run all test envs at once, just run:<br>
+```bash
+tox
+```
+
+tox creates virtual environment for every tox environment defined in
+[tox.ini](tox.ini). Create and activate a virtualenv with the development requirements, i.e.:
+
+```bash
+source .tox/unit/bin/activate
+```
+
+## Integration tests
+TBD
```

Comment:
I've run the file through [Prettier](https://prettier.io/), I think it looks better
```suggestion
## Developing and testing

Testing of the Snap is done using `tox`. To run tests, from this repository's main directory
execute one of the following commands:

```shell
tox -e lint      # code style
tox -e static    # static analysis
tox -e unit      # unit tests
```

To run all test envs at once, just run:<br>

```bash
tox
```

tox creates virtual environment for every tox environment defined in
[tox.ini](tox.ini). Create and activate a virtualenv with the development requirements, i.e.:

```bash
source .tox/unit/bin/activate
```

## Integration tests

TBD
```

Reply:
Nice. Thanks!